Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -630,6 +630,20 @@ public extension Purchases {
         return try await logOutAsync()
     }
 
+    ///
+    /// Updates the current appUserID to a new one, without associating the two.
+    /// This method is **only available** in Custom Entitlements Computation mode.
+    /// Receipts posted by the SDK to the RevenueCat backend after calling this method will be sent
+    /// with the newAppUserID.
+    /// 
+    func switchUser(newAppUserID: String) {
```

Comment:
Ditto:
```suggestion
    func switchUser(to newAppUserID: String) {
```

Will this be available to `Obj-C`? It'll probably need a custom name too.